### PR TITLE
Revert RocksDB backend settings:

### PR DESCRIPTION
**Pull request by vinniefalco**
This reverts the change that makes RocksDBQuick the default settings for
node_db "type=rocksdb". The quick settings can be obtained by setting
"type=rocksdbquick".

RocksDBQuick settings are implicated in memory over-utilization problems
seen recently.

### CODE REVIEW

File: src/ripple/nodestore/backend/RocksDBFactory.cpp [hunk 1]
```diff
@@ -352,11 +358,6 @@ class RocksDBBackend
     {
         storeBatch (batch);
     }
-
-    void
-    verify() override
```
Comment (scottschurr) [inline src/ripple/nodestore/backend/RocksDBFactory.cpp]: Just checking that losing this `verify() override` is correct.  We want the behavior that comes from the base class?  I'll assume so...

Comment (vinniefalco) [inline src/ripple/nodestore/backend/RocksDBFactory.cpp]: Something is wrong... `verify` should be present. I wonder how this compiles...

**Comment by scottschurr** [comment]
Looks like legit build errors from Travis:

```
src/beast/beast/nudb/detail/posix_file.h:78:18: error: no member named
      'strerror' in the global namespace
        return ::strerror(errnum);
```
```
In file included from src/beast/beast/nudb/tests/callgrind_test.cpp:20:
In file included from src/beast/beast/nudb/store.h:53:
src/beast/beast/cxx14/memory.h:33:37: error: allocating an object of abstract
      class type 'ripple::NodeStore::RocksDBBackend'
    return std::unique_ptr <T> (new T (std::forward <Args> (args)...));
```

**Comment by vinniefalco** [comment]
@scottschurr Should be fixed now

**Comment by scottschurr** [comment]
Are we really sure we want to include the `crawl` feature in a commit that targets fixing server problems?  It feels to me like that belongs in a separate commit.  But I'm not in on all business decisions.

**Comment by vinniefalco** [comment]
@scottschurr The server issue was hotfixed in *0.27.0-sp1*. This just brings the change into _develop_.

**Comment by scottschurr** [comment]
Ah, I get it.  Okay.

**Comment by scottschurr** [comment]
Is there an issue withe the vcproj files?  Or the vcproj file change test?  Travis has the following odd failure:

```
The command "scons vcxproj" exited with 0.
$ (git status --porcelain Builds/VisualStudio2013/RippleD.vcxproj | grep -E '^.+M' -q && exit 1 || exit 0)
```
Then is stopped dead.

**Comment by scottschurr** [comment]
If I had to guess, perhaps the `exit`s in the following line are kicking too hard?
```
(git status --porcelain Builds/VisualStudio2013/RippleD.vcxproj | grep -E '^.+M' -q && exit 1 || exit 0)
```

**Comment by miguelportilla** [comment]
vcxproj files seem okay. This builds fine in Windows. scons vcxproj did not produce different results.

**Comment by scottschurr** [comment]
I'm looking for a way to patch up Torrie's change, but I'm not current with the UNIX command line so it's taking awhile.

**Comment by miguelportilla** [comment]
The reverted changes look okay to me.

The backend.test and database.test changes were not reverted. Was that intentional?

**Comment by vinniefalco** [comment]
@miguelportilla Yes. The tests should use the original settings, not the quick one. Although its not likely to matter for the tests.

**Comment by scottschurr** [comment]
Scott Determan is looking into a fix for the `scons vcxproj` Travis failure.

**Comment by vinniefalco** [comment]
Travis works now, I yanked the Travis changes out.

**Comment by scottschurr** [comment]
:+1:

**Pull request merged**